### PR TITLE
Pref UI Fix

### PR DESCRIPTION
**Pull request by Redeem-Grimm-Satoshi**
1. Theme applied to pref ui dialogs

### CODE REVIEW

**Review by wbrunette — CHANGES_REQUESTED** [review]

File: androidlibrary_lib/src/androidTest/java/org/opendatakit/provider/fragment/AlertNProgessMsgFragmentMgerTest.java [hunk 1]
```diff
@@ -14,6 +14,8 @@
 import androidx.fragment.app.Fragment;
 import androidx.fragment.app.FragmentManager;
 import androidx.test.core.app.ActivityScenario;
+
```
[the remark this answers is not in the document]
Comment (Redeem-Grimm-Satoshi) [reply androidlibrary_lib/src/androidTest/java/org/opendatakit/provider/fragment/AlertNProgessMsgFragmentMgerTest.java:17]: It was moved to the provider package, I guess the previous developer created a separate package.

Still, I'll need further clarification if it's not what you mean or If I don't understand at all

[the remark this answers is not in the document]
Comment (Redeem-Grimm-Satoshi) [reply androidlibrary_lib/src/androidTest/java/org/opendatakit/provider/fragment/AlertNProgessMsgFragmentMgerTest.java:17]: @wbrunette

[the remark this answers is not in the document]
Comment (wbrunette) [reply androidlibrary_lib/src/androidTest/java/org/opendatakit/provider/fragment/AlertNProgessMsgFragmentMgerTest.java:17]: This shows your code change is moving it to the provider package. Which is why it requires the input. Maybe you accidentally dragged it in Android Studio?

[the remark this answers is not in the document]
Comment (Redeem-Grimm-Satoshi) [reply androidlibrary_lib/src/androidTest/java/org/opendatakit/provider/fragment/AlertNProgessMsgFragmentMgerTest.java:17]: @wbrunette No I didn't move it to the provider package intentionally

[the remark this answers is not in the document]
Comment (wbrunette) [reply androidlibrary_lib/src/androidTest/java/org/opendatakit/provider/fragment/AlertNProgessMsgFragmentMgerTest.java:17]: please move it back :) as you can see the location in the branch you are merging too has it not in provider. https://github.com/odk-x/androidlibrary/blob/outreachy-2023/androidlibrary_lib/src/androidTest/java/org/opendatakit/fragment/AlertNProgessMsgFragmentMgerTest.java

[the remark this answers is not in the document]
Comment (wbrunette) [reply androidlibrary_lib/src/androidTest/java/org/opendatakit/provider/fragment/AlertNProgessMsgFragmentMgerTest.java:17]: this is why code reviews are done is catch things like this.

[the remark this answers is not in the document]
Comment (Redeem-Grimm-Satoshi) [reply androidlibrary_lib/src/androidTest/java/org/opendatakit/provider/fragment/AlertNProgessMsgFragmentMgerTest.java]: @wbrunette I just refactored the code, the changes are located in the new commit

[the remark this answers is not in the document]
Comment (Redeem-Grimm-Satoshi) [reply androidlibrary_lib/src/androidTest/java/org/opendatakit/provider/fragment/AlertNProgessMsgFragmentMgerTest.java]: Moved Fragment package back to it's original location

**Pull request merged**